AndroidUI/Utils/DriverExtensions.cs

```diff
@@ -25,11 +27,51 @@ public static void PressBack()
             driver.Navigate().Back();
         }
 
-        public static void PressKey(int AndroidKeyCode) 
+        public static void PressKey(int AndroidKeyCode)
         {
             //Use AndoidKeyCode.key to find key code
             driver.PressKeyCode(AndroidKeyCode);
         }
 
+        public static void ScrollToElement(By element, ScrollDirection direction)
+        {
+            bool elementFound = false;
+            while (!elementFound)
+            {
+                try
+                {
+                    var result = driver.FindElement(element);
+
+                    if (result != null)
+                    {
+                        elementFound = true;
+                    }
+                }
+                catch (Exception)
```

Comment:
1.This approach will not pass at CI as it's depend on monitor resolution
2. Better to try to find element at first above then go down
3. As I mentioned previously, please add for PR only code which logically depend on name of it